NuGet/NuGet.Jobs
# Merge NuGet.Services.Metadata into NuGet.Jobs

Address https://github.com/NuGet/Engineering/issues/2669.

The approach taken was to add NuGet.Services.Metadata as a remote on my local NuGet.Jobs clone, then merge NuGet.Services.Metadata's `dev` branch into NuGet.Jobs `dev` branch, allowing unrelated histories.

The merge conflicts were essentially places where both  repositories had files with the same name. I logically merged the files that have the same name and merged the NuGet.Services.Metadata projects into NuGet.Jobs.sln.

All commits before merge commit 2fe83de7da67c3776005aebac7d417a71c91b3f6 are the originals from NuGet.Services.Metadata. Everything afterwards is reviewable on a commit-to-commit basis.

I plan on **NOT** squashing so we can retain history on a per-file basis.

![image](https://user-images.githubusercontent.com/94054/89088151-b1f73e80-d34b-11ea-8457-fcdc89180fd5.png)

Follow-up tasks:
1. Test build and deployment:
    - Azure Search Service + Azure Search functional tests
    - Catalog2AzureSearch
1. Update Azure Search Service release build to pull from NuGet.Jobs (this is a change to NuGetDeployment)
1. Update release pipelines for NuGet.Services.Metadata jobs to pull artifacts from NuGet.Jobs builds
1. Delete NuGet.Services.Metadata CI and release build
1. Disable CI gate on NuGet.Services.Metadata
1. Make a commit to NuGet.Services.Metadata deleting all content and giving a "moved" notice.
1. Archive NuGet.Services.Metadata
